sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM rust:1.32 as base-build
+
+ENV TMP_BUILD_DIR=/tmp/build
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
+ENV RUST_VERSION=1.32.0
+ENV PATH="$CARGO_HOME/bin:$PATH"
+
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl \
+        make \
+        musl-tools \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
+        sudo \
```

Comment:
Why do you need sudo?

Reply:
because the `cri.sh` script uses sudo to mount the root image to install the agent and all that jazz.

Reply:
But you're running as root in the container, no?

Reply:
Yea, turns out I added sudo due to the CRI install scripts using sudo. To which, if it isn't installed, fails.

Reply:
This stage of the Dockerfile is not used to build CRI or containerd.